### PR TITLE
Comments failed buildimages tests for arm

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
 /kind test
 /kind failing-test
> /kind feature

**What this PR does / why we need it**:
There's something wrong with the docker build in arm platform, so comments them first for workaround.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
Please add `TODO:` tag in comments when can we uncomment this.

**Comment by fisherxu** [comment]
> Please add TODO: tag in comments when can we uncomment this.
Otherwise we shall just remove them.

We need them for arm test, and I have added TODO: tag in comments, ptal. @kevin-wangzefeng @chendave

File: .travis.yml [hunk 1]
```diff
@@ -101,30 +101,31 @@ jobs:
...
       name: "build edgesite image on amd64"
-    - script: make edgesiteimage ARCH="arm64v8"
-      arch: arm64
-      name: "build edgesite image on arm64"
     - script: make bluetoothdevice_image
       arch: amd64
       name: "build bluetoothdevice image on amd64"
     - script: make bluetoothdevice_image
       arch: arm64
       name: "build bluetoothdevice image on arm64"
```
Comment (chendave) [inline .travis.yml]: the image is not actually build, there are some issues in the Makefile, I will propose a patch to fix it after this got merged.

File: .travis.yml [hunk 2]
```diff
@@ -101,30 +101,31 @@ jobs:
...
+#      name: "build cloudimage on arm64"
+#    - script: make admissionimage
+#      arch: arm64
+#      name: "build admission image on arm64"
+#    - script: make edgeimage ARCH="arm64v8"
+#      arch: arm64
+#      name: "build edge image on arm64"
+#    - script: make edgesiteimage ARCH="arm64v8"
+#      arch: arm64
+#      name: "build edgesite image on arm64"
```
Comment (chendave) [inline .travis.yml]: I have raised a issue in the travis community, hopefully, we cannot get some answer soon.

**Comment by kevin-wangzefeng** [comment]
/approve
Thanks

File: .travis.yml (hunk 2 above)
Comment (fisherxu) [reply .travis.yml]: OK, thanks!

**Pull request merged**